src/main/java/mate/academy/springboot/swagger/service/SortingService.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.springboot.swagger.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public class SortingService {
+    public static List<Sort.Order> parseSortingOrders(String sortBy) {
+        List<Sort.Order> orders = new ArrayList<>();
+        if (sortBy.contains(":")) {
```

Comment:
And also each magic value is better to make constant, especially ":" that you use twice. You can name it DELIMITER for example